include/benchmark/benchmark.h

```diff
@@ -1122,7 +1129,10 @@ class BENCHMARK_EXPORT Benchmark {
   AggregationReportMode aggregation_report_mode_;
   std::vector<std::string> arg_names_;       // Args for all benchmark runs
   std::vector<std::vector<int64_t> > args_;  // Args for all benchmark runs
+
   TimeUnit time_unit_;
+  bool use_default_time_unit_;
```

Comment:
I thought we agreed that this was not needed now that `time_unit_` is init'd to the `GetDefaultTimeUnit()`?

Reply:
If you create global test objects (like how it is shown in the examples with the helper macros) then `time_unit_` is initialized before the arguments are parsed.